db/migrate/20181008141541_change_optional_to_required_for_related_services.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeOptionalToRequiredForRelatedServices < ActiveRecord::Migration[5.2]
+  def change
+    ServiceRelation.all.each do |service_relation|
+      if service_relation.optional == true
```

Comment:
This block could be shortened to `service_relation.update_attribute(:optional, !service_relation.optional)`